homeassistant/components/universal/media_player.py

```diff
@@ -80,7 +80,6 @@
 CONF_ATTRS = "attributes"
 CONF_CHILDREN = "children"
 CONF_COMMANDS = "commands"
-CONF_SERVICE = "service"
 CONF_SERVICE_DATA = "service_data"
```

Comment:
It looks like CONF_SERVICE_DATA might be unused too?

Reply:
Ok. Found another one, too.